components/gitpod-db/go/oidc_client_config.go

```diff
@@ -30,6 +29,9 @@ func (c *OIDCClientConfig) TableName() string {
 	return "d_b_oidc_client_config"
 }
 
+type OIDCSpec struct {
```

Comment:
I think the naming is misleading, as it's to far away from "configuration of a client". In case that would follow the k8s slang, let's not do this 🙏🏻